content/topics/kotlin/classes/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Classes 
+ready: true
+prerequisites:
+  hard: ["topics/kotlin/basic-control-flow"]
+  soft: []
+
+---
+## Classes
```

Comment:
I'm not 100% convinced that you looked at how this content would render. The html would start off looking like this:

```
<h1>Classes</h1>
<h2>Classes</h2>
```
This is redundat redundant

Reply:
Oh I totally forgot about that little detail. I did check just dint notice.